Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -130,18 +132,41 @@ extension Google_Protobuf_Duration: _CustomJSONCodable {
     }
 }
 
-
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
 
     public init(floatLiteral value: Double) {
+        // TODO: this rounds the nanos part towards zero, not to the nearest
```

Comment:
And also right now if they ask for the literal conversion they get (1, 999999999) but if they use the TimeInterval init they get (2, 0)--the combination of which is at least very hard to explain.